### ISSUE TITLE
Can not install .cap on ACS Acosj card

### ISSUE THREAD

**Issue by hoody424** [darconeous/gauss-key-card#6]
Howdy,

reading trough your "working cards" guide, i ended up buying a ACS Acosj Contactless Card from Alza.de (https://www.alza.de/acs-acosj-java-card-contactless-d7298779.htm). According the distributor this card has ISO-14443 A+B support. 
ACS offers their own tool for uploading .cap files (https://www.acs.com.hk/en/products/407/acosj-java-card-contactless/).
I am using my rooted google pixel 8pro with the app Smart Card Reader Pro together with vsmartcard0.9 as the NFC reader. 
Card connection seems fine, also with GlobalPlatformPro (gp.exe -l succesfully listing the card).
I was not able yet to upload an .cap file, either with the provided tool or with GlobalPlatformPro. 
Unfortunately i get following error:

Virtual Smart Card Architecture Virtual PCD 0
Card reset success
ATR: 3B8180012828
time: 212 ms
;.SET-KEY 00/404142434445464748494A4B4C4D4E4F/404142434445464748494A4B4C4D4E4F/404142434445464748494A4B4C4D4E4F
;Successful to set the key
;Select issure secure domain
<= 00A4040000 (9000)
=> 6F5C8408A000000151000000A550734A06072A864886FC6B01600C060A2A864886FC6B02020201630906072A864886FC6B03640B06092A864886FC6B040255650B06092A864886FC6B020103660C060A2B060104012A026E01039F6501FF(9000)
time: 103 ms
;Authentication
<= 8050000008 1122334455667788 <C,RFU,0>(9000)
=> 0000026501830395366220020026B9E3E24BF377B7FA582D12B0A01D(9000)
time: 96 ms
Session Key(Enc) : CA6E2792A12108A8D9F789C52E0A86D5
Session Key(Cmac): 5F3F277DED4839E87A4CEDF32B44052F
Session Key(Rmac): 15E580F5289B40086AA83BB230D155FD
Session Key(Dek) : 07D7AEBBF21DC0A00F7A0B03C2FE41FD

;8482000010 <F,RFU,0>(9000)
<= 848200001075842C67A74854895A51CFD391C8F08E
=> (9000)
time: 92 ms
;;.AUTH 00 plain
;Delete original applets
<= 80E4008011 4F0FF465736C614C6F6769633030325801
=> (6A88)
time: 164 ms
<= 80E4008010 4F0EF465736C614C6F67696330303258
=> (6A88)
time: 109 ms
;start upload cap file
<= 80E6020013 0EF465736C614C6F6769633030325800000000(9000)
=> 00(9000)
time: 123 ms
<= 80E80000FF C482066B010018DECAFFED01020401010EF465736C614C6F6769633030325802001F0018001F0013002900B60016025700DA005A00000183000A000500C104010004002904000107A0000000620001030107A0000000620102030107A0000000620101030107A0000000620201030013010FF465736C614C6F6769633030325801009106001600800000FF000100000000820305000507010000010B0702570200CF800E00DF001000E0800E00F000100110188C00137A0410187B0016037B0016928E04001A00187B001C037B001C928E04001A03187B001E037B001E928E04001A04187B0005037B0005928E04001A05187B0006037B0006928E04001A06(9000)
(
=> )
time: 585 ms
Error: ? - 31

**Comment by darconeous** [comment]
Have you tried uploading any other applets? Would be good to know if there was something wrong with GaussKeyCard or if there is just something wrong with your process. 

By the way… Lots of smart card chips support both 14443A and 14443B,  but they can only support one or the other. Which one is usually permanently set during prepersonalization. But I think your card is set up for 14443A, which is good. 

I’m curious if you will be able to get it to work with this card!

**Comment by hoody424** [comment]
I just checked and got same error. I tried [openjavacard-ndef](https://github.com/OpenJavaCard/openjavacard-ndef)
Gp returns LOAD failed: 0x643A. Strange that i can not even find this APDU error in a list. 
Before executing the CAP-Tool is asking me following:
![acscaptool1](https://github.com/darconeous/gauss-key-card/assets/67396028/8623c38b-8ca9-4c75-937a-d6b1b478209f)


Any suggestions?

**Comment by herrfrei** [comment]
I have the same card and installed the .cap file successfully using a ACR 1252U USB NFC Reader III. The card works well in my Tesla car.

**Comment by hoody424** [comment]
> I have the same card and installed the .cap file successfully using a ACR 1252U USB NFC Reader III. The card works well in my Tesla car.

Which Loader has been used? Same as shown in my Screenshot?

**Comment by herrfrei** [comment]
No, I used [GlobalPlatformPro](https://github.com/martinpaljak/GlobalPlatformPro) as given in the readme:

```
$ gp -install bin/GaussKeyCard.cap
```

**Comment by ZLG77** [comment]
<img width="1337" height="825" alt="Image" src="https://github.com/user-attachments/assets/b21444ad-b18d-4148-bdde-febd46872c17" />

**Comment by ZLG77** [comment]
I can successfully install GaussKeyCard.cap using NXP's J3R180 and J3R452.

<img width="800" height="800" alt="Image" src="https://github.com/user-attachments/assets/934132db-d8da-466b-bf7c-d287f8b76618" />

**Comment by ZLG77** [comment]
We can provide batch customization of cards in various styles.If you need customization, please open the link below to contact our sales staff.

https://www.smartcardrfidtag.com/
